com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+                    "url" : "https://excellmedia.dl.sourceforge.net/project/libwpg/libwpg/libwpg-0.2.2/libwpg-0.2.2.tar.xz",
+                    "sha256" : "b49bfd77b9447534dee0d43bbdc48b4dcb09339605be21c01726243d5069a6cd"
+                }
+            ]
+        },
+        {
+            "name" : "psiconv",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://www.frodo.looijaard.name/system/files/software/psiconv/psiconv-0.9.9.tar.gz",
+                    "sha256" : "6d51fe79b502a1e277bea275a574ae2db5b1b9d7daef703a8fa3635ae02a8bb0"
+                }
+            ]
+        },
+        {
+            "name" : "libgda"
+        },
+        {
+            "name" : "libgnomedb"
+        },
+        {
+            "name" : "gtkmathview",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://helm.cs.unibo.it/mml-widget/sources/gtkmathview-0.8.0.tar.gz",
+                    "sha256" : "1dc30175da6a3c560a7d62d1abe1c2f9829d988e6f1a7c5e766544575c558c43"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/0003-gcc-4.3-build-fixes.patch"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/0004-gcc-4.4-build-fixes.patch"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/0005-gcc-4.7-build-fixes.patch"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/missingLib.diff"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/remove_t1_include.diff"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/fix-cpp-headers.patch"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "patches/gtkmathview/gcc-6-build-fixes.patch"
+                }
+            ]
+        },
+        {
+            "name" : "libwps",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://newcontinuum.dl.sourceforge.net/project/libwps/libwps/libwps-0.2.10/libwps-0.2.10.tar.xz",
+                    "sha256" : "2d9771489d2474d483089174e50f57f35785d0a444d390b0dfc7eae1b42b4b12"
+                }
+            ]
+        },
+        {
+            "name" : "abiword",
+            "config-opts" : [
+                "--enable-plugins=auto"
+            ],
+            "sources" : [
+              {
+                "type" : "git",
+                "url" : "https://github.com/AbiWord/abiword",
+                "branch": "ABI-3-0-0-STABLE"
```

Comment:
flathub releases need to be pinned to a specific commit as well, for reproducibility.

Reply:
Added the commit.